source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -80,17 +80,15 @@ OpenGLContext: class extends GpuContext {
 	drawQuad: func { this _renderer drawQuad() }
 	drawLines: func (pointList: VectorList<FloatPoint2D>, projection: FloatTransform3D, pen: Pen) {
 		positions := pointList pointer as Float*
-		this _linesShader projection = projection
 		this _linesShader add("color", pen color normalized)
-		this _linesShader use(null)
+		this _linesShader use(null, projection, FloatTransform3D identity)
```

Comment:
Now these calls don't even have to perform any matrix multiplications on the CPU.